pom.xml

```diff
@@ -71,25 +116,121 @@
         </executions>
       </plugin>
       <plugin>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.1</version>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-release-plugin</artifactId>
+        <version>3.0.0-M6</version>
+        <configuration>
+          <useReleaseProfile>false</useReleaseProfile>
+          <releaseProfiles>cloudbees-release</releaseProfiles>
+        </configuration>
```

Comment:
I suspect we could add anything needed (like GPG signing) to the standard `release` profile, but this is fine too.